extensions/integration-tests/src/stressutils.ts

```diff
@@ -0,0 +1,156 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/*
+	This contains all the definitions for Stress decorators and the utility functions and definitions thereof
+*/
+import 'mocha';
+import { AssertionError } from 'assert';
+import { getSuiteType, SuiteType } from './fmkUtils';
+import assert = require('assert');
+
+export interface StressOptions {
+	runtime?: number;
+	dop?: number;
+	iterations?: number;
+	passThreshold?: number;
+}
+
+const DefaultStressOptions: StressOptions = { runtime: 120, dop: 4, iterations: 10, passThreshold: 0.95 };
+
+// This simulates a sleep where the thread is suspended without spinning for a given number of milliseconds before resuming
+//
+export function sleep(ms: number) {
```

Comment:
please add return types to methods, particularly if they have `return` statements.